src/site/content/en/blog/defining-core-web-vitals-thresholds/index.md

```diff
@@ -0,0 +1,486 @@
+---
+title: Defining the Core Web Vitals metrics thresholds
+subhead: The research and methodology behind Core Web Vitals thresholds
+authors:
+  - bmcquade
+description: The research and methodology behind Core Web Vitals thresholds
+date: 2020-05-21
+updated: 2020-05-21
+hero: hero.png
+alt: The research and methodology behind Core Web Vitals thresholds
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - performance
+  - web-vitals
+---
+
+[Core Web Vitals](/vitals/#core-web-vitals) are a set of field metrics that
+measure important aspects of real-world user experience on the web. Core Web
+Vitals includes metrics, as well as target thresholds for each metric, which
+help developers qualitatively understand whether the experience of their site is
+good, needs improvement, or is poor. This post will explain the approach used to
```

Comment:
```suggestion
"good", "needs improvement", or is "poor". This post will explain the approach used to
```
Should we consistently use quotes when referring to these keywords? We seem to use quotes consistently throughout the rest of the post.